### ISSUE TITLE
Update to Serde 0.9

### ISSUE THREAD

**Issue by indiv0** [indiv0/wolfram-alpha-rs#18]
This is currently blocked by https://github.com/serde-rs/xml/issues/35.

**Comment by RReverser** [comment]
Would you like to try with https://github.com/RReverser/serde-xml-rs perhaps? Just published it today after developing and testing on few quite complex XML structures internally.

**Comment by indiv0** [comment]
Yeah, sure. I'll port it over in the next few days. Luckily I have decently comprehensive test cases so any discrepancies should be found quickly :)
